### PR TITLE
Adds a new page /geo that shows realtime traffic around the world

### PR DESCRIPTION
**Pull request by snatchev**
screen shot 2017-01-10 at 11 37 40 am

### CODE REVIEW

File: app/models/bacon.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-class Bacon < ActiveRecord::Base
-  
+class Bacon < ApplicationRecord
```
Comment (KrauseFx) [inline app/models/bacon.rb:1]: Why this change?

File: app/models/application_record.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  self.abstract_class = true
```
Comment (KrauseFx) [inline app/models/application_record.rb:2]: What's that class for?

File: app/controllers/update_checks_controller.rb [hunk 3]
```diff
@@ -233,6 +234,10 @@ def unique
   end
 
   private
+    def geo_code
+      GeoCoder.broadcast(request.remote_ip)
```
Comment (KrauseFx) [inline app/controllers/update_checks_controller.rb:240]: Maybe add a comment that the IP address is never stored anywhere, and only used to detect the country to show on the /geo earth visualisation.

File: app/models/geo_coder.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
+class GeoCoder
+
```
Comment (KrauseFx) [inline app/models/geo_coder.rb:7]: Add a comment here on what this class is used for 👍

File: app/models/bacon.rb (hunk 1 above)
Comment (snatchev) [reply app/models/bacon.rb:1]: Upgrade to Rails 5
http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#active-record-models-now-inherit-from-applicationrecord-by-default

File: app/models/application_record.rb (hunk 2 above)
Comment (snatchev) [reply app/models/application_record.rb:2]: Same. Rails 5 requirement

**Review by KrauseFx — APPROVED** [review]
